test/kubemark/start-kubemark.sh

```diff
@@ -221,7 +221,7 @@ function create-and-upload-hollow-node-image {
 function create-and-upload-hollow-node-image-bazel {
   RETRIES=3
   for attempt in $(seq 1 ${RETRIES}); do
-    if ! bazel run //cluster/images/kubemark:push --define PROJECT="${PROJECT}"; then
+    if ! bazel run //cluster/images/kubemark:push --define REGISTRY="${FULL_REGISTRY}"; then
```

Comment:
`if ! bazel run //cluster/images/kubemark:push --define REGISTRY="${REGISTRY}" --define REPO_PATH_BASE ="${REPO_PATH_BASE}"; then`

Reply:
I prefer it to be simpler - one variable.

I'll leave refactoring of this to the team that owns it ( @shyamjvs and @wojtek-t can overrule if needed)